lib/semian/atomic_enum.rb

```diff
@@ -0,0 +1,37 @@
+module Semian
+  class AtomicEnum < AtomicInteger #:nodoc:
+    undef :increment_by
+    undef :increment
```

Comment:
Why `undef` instead of overriding them with an exception? Does it actually matter that much even if they're defined? It'd work and would be equivalent to iterating on the enum, which is not a crazy thing to do.